recipe-box-frontend/src/main/frontend/src/components/RecipeView/RecipeView.tsx

```diff
@@ -0,0 +1,26 @@
+import Header from "../Header/Header";
+import Footer from "../Footer/Footer";
+import {useState} from "react";
+import Categories from "../RecipesList/Categories";
+import "./RecipeView.css";
+import RecipeViewGrid from "./RecipeViewGrid";
+
+
+export default function RecipeView() {
+    const [mealType, setMealtype] = useState("");
+    const [searchQuery, setSearchQuery] = useState("");
+
+    return (
+        <div className="recipe-view-container">
+            <Header/>
+            <main>
+                <Categories
```

Comment:
P.S.: Since we are including the Categories component here, shouldn't it take us back to the RecipesList and filter accordingly when clicked?

Reply:
It should indeed redirect back to the RecipesList. We can discuss the Categories component's presence in the next meeting, but until then I think it's best we remove it from here and create an additional task to add it and modify it to work on this page as well if merged in the meantime

Reply:
I have already made the layout to include the category menu, so I'll leave it as is for now. This layout is also used by the recipe list page, so if I remove the  category menu from the layout, it will also be removed from the recipe list page and the recipe view page.